app.js

```diff
@@ -0,0 +1,72 @@
+const tipAmount = document.querySelector(".tip-amount");
+const totalAmount = document.querySelector(".total-amount");
+const inputBill = document.querySelector(".bill");
+const selectBtn = document.querySelectorAll("button");
+const peopleNum = document.querySelector(".number");
+const resetBtn = document.querySelector(".reset-btn");
+const ifZero = document.querySelector(".if-zero");
+const customTip = document.querySelector(".custom");
+let selectTip;
+
+inputBill.addEventListener("click", () => {
+  inputBill.style.outlineColor = "hsl(185, 41%, 84%)";
+});
+
+peopleNum.addEventListener("click", () => {
+  peopleNum.style.outlineColor = "hsl(185, 41%, 84%)";
+});
+
+function resetBtnColor() {
+  for (let i = 0; i < Object.keys(selectBtn).length; i++) {
+    selectBtn[i].classList.remove("clicked");
+  }
```

Comment:
Javascript에서 제공하는 api를 사용해봐도 좋을 것 같습니다!

```suggestion
const selectBtnsKey = Object.keys(selectBtn) ;
selectBtnsKey.forEach((element) => {
 // ... some logic 
 })
```